project/src/main/world/creature/creature-loader.gd

```diff
@@ -8,6 +8,8 @@ class converts that information into granular information such as 'her Eye/Sprin
 loads resource files specific to each creature.
 """
 
+const INSTRUCTOR_PATH := "res://assets/main/creatures/instructor/creature.json"
```

Comment:
comment

Reply:
Added comment, `the creature definition path for the instructor who leads tutorials`